regression_tests/move_output.jl

```diff
@@ -46,3 +46,34 @@ if get(ENV, "BUILDKITE_PIPELINE_SLUG", nothing) == "climaatmos-ci"
 else
     @info "ENV keys: $(keys(ENV))"
 end
+
+get_ref_counter(p) = parse(Int, first(readlines(joinpath(p, "ref_counter.jl"))))
+function cleanup_central(cluster_data_prefix)
+    @warn "Cleaning up old files on central"
+    # Get (sorted) array of paths, `pop!(sorted_paths)`
+    # is the most recent merged folder.
+    sorted_paths = sorted_dataset_folder(; dir = cluster_data_prefix)
+    if !isempty(sorted_paths)
+        N = length(sorted_paths) - 10
```

Comment:
Maybe adding something like this to make clear what 10 and 5 represent?
```suggestion
keep_latest_n = 10
keep_latest_ref_counters = 5
```